EntAppProject/src/main/java/com/jobook/EntAppProject/JobBookController.java

```diff
@@ -1,18 +1,33 @@
 package com.jobook.EntAppProject;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 
+import com.jobook.EntAppProject.Services.UserServiceStub2;
+import com.jobook.EntAppProject.dto.UserDTO;
+
 @Controller
 public class JobBookController {
 	
+	@Autowired
+	private UserServiceStub2 userServiceStub2;

```

Comment:
Why does the variable name have the "2" in it?  That often is a sign that it's not named to describe what it does... or there is some legacy technical debt floating around in UserServiceStub, UserServiceStub3, etc.  I recommend cleaning this up so that it is more intuitive, less confusing.